src/main/java/frc/robot/Constants.java

```diff
@@ -18,13 +18,19 @@ public final class Constants {
   public static final int FLIGHTSTICKUSBINDEX = 1;
   /// CAN Bus Devices
   // Drive Train Motors
-  public static final int MOTORFRONTRIGHTID = 1;
-  public static final int MOTORBACKRIGHTID = 2;
-  public static final int MOTORFRONTLEFTID = 3;
-  public static final int MOTORBACKLEFTID = 4;
+  public static final int MOTORFRONTLEFTID = 11;
+  public static final int MOTORBACKLEFTID = 12;
+  public static final int MOTORFRONTRIGHTID = 13;
+  public static final int MOTORBACKRIGHTID = 14;
+  // Arm movement motors
+  public static final int ARMXMOTORID = 21;
+  public static final int ARMYMOTORID = 22;
+  public static final int ARMCLAWMOTORID = 23;
   // Ultrasonic Sensors and ports.
   public static final int ULTRASONIC1PORT = 0;
   // Buttons
   public static final int SWAPCAMBUTTON = 1;
   public static final int AIMBUTTON = 12;
+  public static final int BALANCEBUTTON = 6;
+  public static final int STRAIGHTBUTTON = 8;
```

Comment:
What does the straight button do?

Reply:
it's used to tell the robot to keep driving straight, just mostly being tested rn.

Reply:
Fair, but can we not use `DriveSubsystem.driveStraight(...)` or the something along those lines? Why create a whole command for it?

Reply:
We could use an instant command, but we might make it more complicated so I prefer using a full command right now.